src/components/select-field/BaseSelectField.js

```diff
@@ -151,6 +178,10 @@ class BaseSelectField extends React.Component<Props, State> {
 
     selectFieldContainerRef: { current: null | HTMLDivElement };
 
+    searchInputRef: React.Element<any>;
```

Comment:
*two* `searchInputRet`s???

Reply:
Addressed offline

Reply:
The second line is needed due to a flow error

Reply:
There should only be one, the second is not needed since HTMLInputElement has a focus attribute.